docs/DocumenterReference.jl

```diff
@@ -0,0 +1,182 @@
+    extras::Vector{Pair{String,DocType}} = Pair{String,DocType}[],
+)
+    config = _Config(current_module, root, subdirectory, extras)
+    pages = Any["Overview"=>"$subdirectory/$current_module.md"]
+    _iterate_over_symbols(config) do key, type
+        push!(pages, Documenter.hide("`$key`" => "$subdirectory/$key.md"))
+        return
+    end
+    push!(CONFIG, config)
+    return pages
+end
+
+function _exported_symbols(mod)
+    contents = Pair{Symbol,DocType}[]
+    for n in names(mod)
+        f = getfield(mod, n)
+        f_str = string(f)
+        if startswith(f_str, "@")
+            push!(contents, n => DOCTYPE_MACRO)
+        elseif startswith(f_str, "Abstract")
+            push!(contents, n => DOCTYPE_ABSTRACT_TYPE)
+        elseif f isa Type
+            push!(contents, n => DOCTYPE_STRUCT)
+        elseif f isa Function
+            if islowercase(f_str[1])
+                push!(contents, n => DOCTYPE_FUNCTION)
+            else
+                push!(contents, n => DOCTYPE_STRUCT)
+            end
+        elseif f isa Module
+            # Skip
+        else
+            push!(contents, n => DOCTYPE_CONSTANT)
+        end
+    end
+    order = Dict(
+        DOCTYPE_MACRO => 1,
+        DOCTYPE_FUNCTION => 2,
+        DOCTYPE_ABSTRACT_TYPE => 3,
+        DOCTYPE_STRUCT => 4,
+        DOCTYPE_CONSTANT => 5,
+    )
+    return sort(contents; by = x -> (order[x[2]], "$(x[1])"))
+end
+
+function _iterate_over_symbols(f, config)
+    current_module = config.current_module
+    for (key, type) in vcat(_exported_symbols(current_module), config.extras)
+        if key isa Symbol
+            doc = Base.Docs.doc(Base.Docs.Binding(current_module, key))
+            if occursin("No documentation found.", string(doc))
+                if type == DOCTYPE_CONSTANT
+                    continue
+                else
+                    error("Documentation missing for $key")
+                end
+            end
+        end
+        f(key, type)
+    end
+    return
+end
+
+function _to_string(x::DocType)
+    if x == DOCTYPE_ABSTRACT_TYPE
+        return "abstract type"
+    elseif x == DOCTYPE_CONSTANT
+        return "constant"
+    elseif x == DOCTYPE_FUNCTION
+        return "function"
+    elseif x == DOCTYPE_MACRO
+        return "macro"
+    elseif x == DOCTYPE_STRUCT
+        return "struct"
+    end
+end
+
+function _add_page(document::Documenter.Document, filename, contents)
+    mdpage = Markdown.parse(contents)
+    document.blueprint.pages[filename] = Documenter.Page(
+        filename, # source,
+        "$(document.user.build)/$filename", # build,
+        "$(document.user.build)/", # workdir,
+        mdpage.content,
+        Documenter.Globals(),
+        convert(MarkdownAST.Node, mdpage),
+    )
+    return
+end
+
+function _build_api_page(document::Documenter.Document, config::_Config)
+    subdir = config.subdirectory
+    overview_md = """
+    # [API](@id DocumenterReference_$(config.current_module))
```

Comment:
You might be getting nonsense edit URL for the pages right now? But you should be able to add a

````markdown
```@meta
EditURL = nothing
```
````

to the top here (or bottom, or wherever, really).

Reply:
Ahhhhh. I knew I was missing something.